### PR TITLE
Adds extension minitest_log

### PR DESCRIPTION
**Pull request by BurdetteLamar**

### CODE REVIEW

File: README.rdoc [hunk 1]
```diff
@@ -677,6 +677,7 @@ minitest-unordered          :: Adds a new assertion to minitest for checking the
                                contents of a collection, ignoring element order.
 minitest-vcr                :: Automatic cassette managment with Minitest::Spec
                                and VCR.
+minitest_log                :: Adds structured logging, data explication, and verdicts.
```
Comment (BurdetteLamar) [inline README.rdoc]: Anything more needed here?
* Gem: https://rubygems.org/gems/minitest_log
* Code: https://github.com/BurdetteLamar/minitest_log

Comment (BurdetteLamar) [reply README.rdoc]: If not to be merged, should I close?

Comment (zenspider) [reply README.rdoc]: cool your jets.

**Comment by zenspider** [comment]
Your project should be named `minitest-log` to fit with rubygems standard naming conventions.

**Comment by zenspider** [comment]
added. thanks!

**Pull request closed without merging**